pyxem/signals/polar_diffraction2d.py

```diff
@@ -304,6 +310,98 @@ def subtract_diffraction_background(
             **kwargs,
         )
 
+    def get_orientation(
+        self,
+        simulation,
+        n_keep=None,
+        frac_keep=0.1,
+        n_best=1,
+        gamma=0.5,
+        normalize_templates=True,
+        **kwargs,
+    ):
+        """Match the orientation with some simulated diffraction patterns using
+        an accelerated orientation mapping algorithm.
+        The details of the algorithm are described in the paper:
+        "Free, flexible and fast: Orientation mapping using the multi-core and
+         GPU-accelerated template matching capabilities in the python-based open
+         source 4D-STEM analysis toolbox Pyxem"
+        Parameters
+        ----------
+        simulation : DiffractionSimulation
+            The diffraction simulation object to use for indexing.
+        n_keep : int
+            The number of orientations to keep for each diffraction pattern.
+        frac_keep : float
+            The fraction of the best matching orientations to keep.
+        n_best : int
+            The number of best matching orientations to keep.
+        normalize_templates : bool
+            Normalize the templates to the same intensity.
+        gamma : float
```

Comment:
Regarding the gamma correction, it would be good to add a reference to https://doi.org/10.1016/j.ultramic.2022.113517 to explain why it is useful?